lib/helpers/getPagesManifest.js

```diff
@@ -1,11 +1,11 @@
 const { join } = require("path");
-const { readJSONSync } = require("fs-extra");
+const { existsSync, readJSONSync } = require("fs-extra");
 const { NEXT_DIST_DIR } = require("../config");
 
 const getPagesManifest = () => {
-  const contents = readJSONSync(
-    join(NEXT_DIST_DIR, "serverless", "pages-manifest.json")
-  );
+  const manifestPath = join(NEXT_DIST_DIR, "serverless", "pages-manifest.json");
+  if (!existsSync(manifestPath)) return {};
```

Comment:
Just to make sure: would `next-on-netlify` still behave correctly even if this function returns an empty object?

Reply:
yes! each place that uses getPagesManifest iterates over it using Object.entries

so it'd be calling `Object.entries({})` which returns an empty array meaning no pages will get included (as appropriate, if there's somehow no pages-manifest) for the relevant page types